app/src/components/keyCloakServiceClientMgr.js

```diff
@@ -168,6 +168,18 @@ class KeyCloakServiceClientManager {
       return undefined;
     }
   }
+
+  async fetchAllClients() {
+
+    log.info('KeyCloakServiceClientManager.fetchAllClients');
```

Comment:
If this needs to be logged, it needs to be at `debug` level instead of info as all this does is say that this function was invoked at some point in the stack.

Reply:
that makes sense